apps/communications/contacts/js/views/list.js

```diff
@@ -1715,9 +1715,8 @@ contacts.List = (function() {
         callback(uuid);
       });
     }
-    }
-    evt.preventDefault();
```

Comment:
It was a mistake, I was going to remove indention and remove the evt.preventDefault. Thanks for pointing this.